apis/src/api/v1/bot/play.rs

```diff
@@ -0,0 +1,64 @@
+use crate::api::v1::auth::auth::Auth;
+use actix_web::get;
+use actix_web::web::{Data, Json};
+use actix_web::{post, web, HttpResponse};
+use anyhow::{anyhow, Result};
+use db_lib::{
+    get_conn,
+    models::{Game, User},
+    DbPool,
+};
+use diesel_async::scoped_futures::ScopedFutureExt;
+use diesel_async::AsyncConnection;
+use hive_lib::State;
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use shared_types::GameId;
+
+#[derive(Serialize, Deserialize)]
+struct PlayRequest {
+    game_id: GameId,
+    piece: String,
+    position: String,
```

Comment:
I suppose that an example of piece/position would be "wP"/"bL-". It would be nicer to expect a standard move ("wP bL-") and cut it yourself

Reply:
Sure, happy to make that change.